index.html

```diff
@@ -1275,7 +1275,71 @@ <h2>The <dfn>InteractionOutput</dfn> interface</h2>
       </ol>
     </section>
   </section>
-  <section> <h3>Using {{InteractionInput}} and {{InteractionOutput}}</h3>
+  
+  <section data-dfn-for="ActionInteractionOutput">
+    <h2>The <dfn>ActionInteractionOutput</dfn> interface</h2>
+    <p>
+      Belongs to the <a>WoT Consumer</a> conformance class.
+      An {{ActionInteractionOutput}} object is always created by the implementations
+      and exposes functionality to interact with long running (asynchronous) actions.<br />
+      Note: a synchronous action MAY not offer any <emph>additional</emph> sensible interaction beyond the ones from {{InteractionOutput}}. 
+    </p>
+    <p>
+      This interface exposes an action status object. Its implementation
+      will allow cancelling asynchronous actions and report the status of a long running action.
+    </p>
+    <pre class="idl">
+      enum ActionStatus {
+        /* "pending", Profile has pending. Not sure what it would mean? */
+        "running",
+        "success", /* Profile uses completed? */
+        "error" /* Profile uses failed? */
```

Comment:
Should we coordinate in some way with the Profile TF to align the terms that are being used? Or is it no problem if they differ, since they are operating on a different level anyway?

Reply:
Good point. Once we feel comfortable with our choices we should definitely reach out. Most of the status terms are 1:1 matches. What seems different is `pending`.

Reply:
FYI: Profile has 4 terms
* `pending` --> we don't have a corresponding term!
* `running `-> same ✅
* `completed `-> we use `success` instead
* `failed `-> we use `error` instead

What can we do with aligning the terms? Use the ones from profile or talk to them? Any opinion?

Reply:
Hmm, their terminology actually sounds quite good to me, so from my side we could also adopt what they are using.

Reply:
I am fine withe the Profile terms also. The question remains if it is fine dropping "pending" then... (same same but different 🙃)
Other opinions?